sys/arm64/arm64/machdep.c

```diff
@@ -226,7 +237,12 @@ set_regs(struct thread *td, struct reg *regs)
 	frame->tf_spsr &= ~PSR_FLAGS;
 	frame->tf_spsr |= regs->spsr & PSR_FLAGS;
 
+#ifdef MORELLO
+	for (i = 0; i < nitems(frame->tf_x); i++)
```

Comment:
```suggestion
	for (int i = 0; i < nitems(frame->tf_x); i++)
```
For loop initializers are fine now (https://reviews.freebsd.org/D25312)

Reply:
I have future changes that switch may of these back to memcpy. We can clean up then if needed.